### PR TITLE
Use specific compose ports for wanda dev/test docker-compose

### PR DESCRIPTION
**Pull request by fabriziosestito**
This PR assigns specific docker-compose forwarded ports to services so they don't clash with the web compose.

It also moves the development phoenix endpoint from 4000 to 4001.

This way, both web and Wanda can be started at the same time.

The checks dev docker-compose remain unchanged.

### CODE REVIEW

**Review by CDimonaco — APPROVED** [review]

File: config/prod.exs [hunk 1]
```diff
@@ -17,6 +17,15 @@ config :wanda, WandaWeb.Endpoint,
 # Do not print debug messages in production
 config :logger, level: :info
 
+config :wanda, Wanda.Messaging.Adapters.AMQP,
```
Comment (arbulu89) [inline config/prod.exs:20]: We don't need to change the prod env, right?
It is already inheriting this exact values from config

**Review by arbulu89 — APPROVED** [review]
Good. I think we don't have to add any change is prod, as it uses the same values as in config

File: config/prod.exs (hunk 1 above)
Comment (fabriziosestito) [reply config/prod.exs:20]: yep exactly. I like to keep the config with "standard" values and the other env as an override of them.

Comment (arbulu89) [reply config/prod.exs:20]: But it is simply overriding the same values XD
So it doesn't do anything at the end.

**Pull request merged**